src_erl/NerlnetApp/src/Source/sourceStatem.erl

```diff
@@ -411,11 +317,11 @@ transmitter(TimeInterval_ms, SourceEtsRef, SourcePid, Epochs ,ClientWorkerPairs,
   end,
   lists:foreach(FuncStart, ClientWorkerPairs),
   TransmissionStart = erlang:timestamp(),
-  case Method of
-    ?SOURCE_POLICY_CASTING_ATOM -> send_method_casting(TransmitterEts, Epochs, TimeInterval_ms, ClientWorkerPairs, BatchesListToSend);
-    ?SOURCE_POLICY_ROUNDROBIN_ATOM -> send_method_round_robin(TransmitterEts, Epochs, TimeInterval_ms, ClientWorkerPairs, BatchesListToSend);
-    ?SOURCE_POLICY_RANDOM_ATOM -> send_method_random(TransmitterEts, Epochs, TimeInterval_ms, ClientWorkerPairs, BatchesListToSend);
-    _Default -> send_method_casting(TransmitterEts, Epochs, TimeInterval_ms, ClientWorkerPairs, BatchesListToSend)
+  case integer_to_list(Method) of % Method is given as an integer
```

Comment:
@GuyPerets106 
This is the bug that was fixed. 
Method was an integer and compared with Atom - therefore, only Default case was chosen.